primevue-gen/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "primvue-gen",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "esbuild": "0.19.5",
```

Comment:
`esbuild` is not a run-time dependency. It is a build-time dependency so we should place it in a `devDependencies` block:
```
  "dependencies": {
    "primevue": "3.40.1"
  },
  "devDependencies": {
    "esbuild": "0.19.5"
  }
```

Reply:
I see you have addressed this commit with 43381cccbaa8ee644d3c72c043c22c552fec6586.

This conversation is now resolved.